### ISSUE TITLE
Ratio of Train set and Test set size 

### ISSUE THREAD

**Issue by abhishekrajgaria** [snap-stanford/neural-subgraph-learning-GNN#4]
As written in common/data.py in load_dataset the ratio of train_set : test_set is 80 : 20,

but as we randomly generate positive and negative query-target pair,
(balance case)  we are getting 4096 graphs, for both training set and testing set,
(imbalance case) we are getting 2048 graphs for both training set and testing set,

So won't that be an issue as the size of train_set is same as test_set?

**Comment by qema** [comment]
Can't confirm right now but I believe there is an option --val_size to change the validation set size. Agree val_size should be increased during a rigorous evaluation
